README.md

```diff
@@ -48,8 +50,14 @@ Entries and their fields are sorted by the `sort` script.
     	author = {Christian K\"{a}stner and Sven Apel and Martin Kuhlemann},
     	title = {{Granularity in Software Product Lines}},
     	booktitle = ICSE,
-    ...
+    	...
+    }
     ```
+- **Month and Year**: 
+  - **Journal Articles**: The publication month and year of a journal is that when the issue (not the article) is published. DBLP seems to have solid knowledge on that (compared to Springer's own website).
+  - **For theses**, we always use the month when the thesis was defended (not when it is submitted and not when it is finally published, which might happen much later).
+  - **For conferences**, the month and year is typically the first day of the conference. As SPLC was in August and September, both months were used.
```

Comment:
I guess this was only true for one specific instance of SPLC? Maybe frame this more like, "Sometimes a conference happens to be at two months, ..."?